build-script.py

```diff
@@ -17,6 +17,16 @@
     WORKSPACE_DIR + '/swift/utils/incrparse/incr_transfer_round_trip.py'
 GYB_EXEC = WORKSPACE_DIR + '/swift/utils/gyb'
 LIT_EXEC = WORKSPACE_DIR + '/llvm/utils/lit/lit.py'
+XC_SETTING = PACKAGE_DIR + '/setting.xcconfig'
```

Comment:
This should be a separate option passed to the build-script from command-line.

Reply:
Done